tests/next_tests/unit_tests/iterator_tests/test_embedded_field_with_list.py

```diff
@@ -0,0 +1,95 @@
+# GT4Py - GridTools Framework
```

Comment:
Maybe a skipped test for the case we don't support yet makes sense: `if(..., const_list_val, list_val)`

Reply:
done